pset-iterators.js

```diff
@@ -107,11 +117,18 @@ between 1850 and 1900. Save the data in a variable called `presidentsBetween1850
 console.log('Problem 3:')
 
 // Add your code below this line
+const presidentsBetween1850and1900 = []
+presidents.forEach((element) => {
```

Comment:
@kyleligman solid use of `.forEach()`, alternatively `.filter()` could have been used here as well